### ISSUE TITLE
alpine as base prevents `exec` using a temporary container

### ISSUE THREAD

**Issue by zramsay** [eris-ltd/eris-keys#45]
for https://github.com/eris-ltd/common/issues/32

(as of may 30):
```
$ docker pull quay.io/eris/keys:latest
$ eris services start keys
```
keys is started... (`eris ls` also looks as it should)
```
$ eris services logs keys
*Starting eris-keys server on 0.0.0.0:4767
```
can't generate a new key
```
$ eris keys gen
Container interactive-af9da86c-cdaf-4397-9d8f-8a9dc388ece8 exited with status 1
```
keys is not running?
```
$ eris services exec keys -ip
bash-4.3#
bash-4.3# eris-keys gen --no-pass
Could not connect to eris-keys server. Start it with `eris-keys server &`. Error: Post http://keys:4767/gen: dial tcp: lookup keys on 8.8.8.8:53: no such host
```
get its ip
```
bash-4.3# cat /etc/hosts
127.0.0.1	localhost
::1	localhost ip6-localhost ip6-loopback
fe00::0	ip6-localnet
ff00::0	ip6-mcastprefix
ff02::1	ip6-allnodes
ff02::2	ip6-allrouters
172.17.0.2	keys b4c0ac9600da keys-435033df-b412-42bc-9778-8f382edac7a6
172.17.0.3	4783eda11569
```
keys is in fact running...
```
bash-4.3# curl 172.17.0.2:4767/gen
{"Response":"","Error":"unexpected end of JSON input"}bash-4.3# 
``` 
double check...
```
bash-4.3# ping keys
PING keys (172.17.0.2): 56 data bytes
64 bytes from 172.17.0.2: seq=0 ttl=64 time=0.194 ms
```
remove all of `/etc/resolve.conf` and replace with only the line `/etc/hosts`
and it works!
```
bash-4.3# eris-keys gen --no-pass
688B36BEC4ED8807CC3052E444928068F4835ED4
```
but not outside the container
```
bash-4.3# exit
$ eris keys gen
Container interactive-69c890d2-87ec-4eea-a271-ef33a32b2df6 exited with status 1
```

**Comment by AFDudley** [comment]
musl and docker weren't playing nice together this was resolved in musl 1.1.13
https://github.com/gliderlabs/docker-alpine/issues/8

**Comment by douglasdzoneericsson** [comment]
I encountered this issue and the resolv.conf hack worked.  Is there a patch that I can apply?

I am running:

Eris 0.12.0
Docker 1.12.1
Ubuntu 16.04 LTS (hosted on VirtualBox 5.1.4 with NATS networking)
